sycl/include/sycl/ext/intel/esimd/memory.hpp

```diff
@@ -7914,33 +7914,47 @@ __ESIMD_API
 
 /// Variant of scatter that uses local accessor as a parameter
 /// template <typename T, int N, int VS = 1, typename AccessorT,
-/// 	  typename PropertyListT = empty_properties_t>
-/// void scatter(AccessorT acc, simd<uint32_t, N / VS> byte_offsets, simd<T, N>
-/// vals, 	simd_mask<N / VS> mask, PropertyListT props = {}); //
-/// (lacc-sc-1)
+/// 	        typename PropertyListT = empty_properties_t>
```

Comment:
nit, here + few more places: 
```suggestion
/// 	      typename PropertyListT = empty_properties_t>
```